halo/app/ready.go

```diff
@@ -13,39 +13,40 @@ type readinessStatus struct {
 	mu                 sync.RWMutex
 	ConsensusSynced    bool    `json:"consensus_synced"`              // Unhealthy if false
 	ConsensusP2PPeers  int     `json:"consensus_p2p_peers"`           // Unhealthy if 0
+	ConsensusHeight    int64   `json:"consensus_height"`              // Unhealthy if 0
+	ConsensusRunning   bool    `json:"consensus_running"`             // Unhealthy if false
 	ExecutionConnected bool    `json:"execution_connected"`           // Unhealthy if false
 	ExecutionSynced    bool    `json:"execution_synced"`              // Unhealthy if false
-	ExecutionP2PPeers  *uint64 `json:"execution_p2p_peers,omitempty"` // Unhealthy if 0
+	ExecutionP2PPeers  *uint64 `json:"execution_p2p_peers,omitempty"` // Unhealthy if 0 (ignored if not available)
+	ExecutionHeight    uint64  `json:"execution_height"`              // Unhealthy if 0
```

Comment:
added Heights, just as a nice to have to users querying the endpoint and looking at the json.